### ISSUE TITLE
XRay traces sent by cloudwatch-agent are missing cloudwatch_logs data

### ISSUE THREAD

**Issue by alex-the-man** [aws/amazon-cloudwatch-agent#1097]
**Describe the bug**
We are using the javascript XRay SDK aws-xray-sdk in our ECS NodeJS backend.
To make sure our ECS Fargate application log shows up on the XRay trace AWS Console page, the XRay traces must contain:
```
[
  {
    "log_group": "<LOG GROUP NAME>",
    "arn": "arn:aws:logs:us-west-2:<ACCT>:log-group:<LOG GROUP NAME>:*"
  }
]
```
To make sure those information are sent, we configured the aws-xray-sdk to include them:
```
export const xRayCloudWatchMetadataPlugin = {
  /**
   * A function to inject required fields for X Ray to parse the application log.
   * @param {function} callback - The callback for the plugin loader.
   */
  getData: (callback: (metadata?: any) => void): void => {
    callback({
      cloudwatch_logs: [
        {
          log_group: environmentVariables.ecsLogGroupName,
          arn: environmentVariables.ecsLogGroupArn,
        },
      ],
    });
  },
  originName: 'AWS::ECS::Container',
};
awsXRay.config([awsXRay.plugins.ECSPlugin, xRayCloudWatchMetadataPlugin]);
```

This works well with the XRay daemon sidecar.
As the latest cloudwatch-agent support XRay as well, we tried to replace the XRay daemon sidecar with cloudwatch-agent.
However, after the switch, traces sent by cloudwatch-agent do not contain the XRay cloudwatch_logs section posted above. And hence the XRay trace pages stopped showing the application log.

**Steps to reproduce**
If possible, provide a recipe for reproducing the error.

**What did you expect to see?**
cloudwatch-agent sent XRay traces with cloudwatch_logs information

**What did you see instead?**
cloudwatch-agent sent XRay logs without cloudwatch_logs information and other plugin information

**What version did you use?**
CWAgent/1.300034.1b536

**What config did you use?**
```
{"agent": {"debug": true},"logs":{"metrics_collected":{"emf":{}}},"traces_collected": {"xray": {}}}
```

**Environment**
OS: Linux
Docker: public.ecr.aws/cloudwatch-agent/cloudwatch-agent:latest

**Additional context**

**Comment by jefchien** [comment]
We've created an issue for this bug https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/31784

**Comment by jefchien** [comment]
The fix was included in the `v1.300037.0` release. Please update and let us know if you're still seeing the issue.
